src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PatientMapper.java

```diff
@@ -71,10 +73,16 @@ private void mapAddress(final UnstructuredAddress unstructuredAddress, final Pat
     }
 
     private void mapNhsNumberIdentifier(final String nhsNumber, final Patient patient) {
-        final Identifier identifier = new Identifier();
-        identifier.setSystem(NHS_NUMBER_SYSTEM);
-        identifier.setValue(nhsNumber);
-        patient.addIdentifier(identifier);
+        patient.addIdentifier()
+            .setSystem(NHS_NUMBER_SYSTEM)
+            .setValue(nhsNumber)
+            .addExtension()
+            .setUrl("https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1")
+            .setValue(new CodeableConcept().addCoding(new Coding()
+                .setSystem("https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-NHSNumberVerificationStatus-1")
+                .setCode("01") //TODO awaiting NHS IOPS to provide answer
+                .setDisplay("Number present and verified")
+            ));
```

Comment:
Formatting is a bit strange to me. I'd format it like this:
```
.setValue(new CodeableConcept().addCoding(
        new Coding()
            .setSystem("https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-NHSNumberVerificationStatus-1")
            .setCode("01") //TODO awaiting NHS IOPS to provide answer
            .setDisplay("Number present and verified")
));
```

Also is getting the value of the code here from the IOPS team still in progress?